src/app/article-list.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { ArticleFormat } from './format';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ArticleListService {
+  public articleListData : ArticleFormat[];
+  constructor() { }
+
+  setArticleList(refrenceObject){
```

Comment:
- `referenceObject` should be renamed to `articleList` or `articleData`
- Mention the type for the function parameter

Reply:
Variable renamed with Meaningful name as suggested and
Parameter type of _functions_ declared